system-build.sh

```diff
@@ -229,6 +229,9 @@ for T in $LLVM_TOOLS; do
 	cp ${TOOL} $LLVM_PREFIX/bin
 done
 
+mkdir -p $LLVM_PREFIX/lib
+cp -Lr $(dirname $(which clang))/../lib/clang/ $LLVM_PREFIX/lib/
+
```

Comment:
I wonder whether symlink or setting C_INCLUDE_PATH would not be enough? I need to check it.